HW3/Menues/Database/Books.cs

```diff
@@ -94,11 +94,11 @@ public void Update(SqlConnection connection)
                 query = $"update Books set name = '{newName}' where name='{nameForUpdate}'";
             }
 
-            using (SqlCommand comm = new SqlCommand(query, connection))
+            using (SqlCommand command = new SqlCommand(query, connection))
```

Comment:
to line 88:
there are three types of empty: null, "", "   ".
for catching all of them use IsNullOrWhiteSpace; IsNullOrEmpty don't catch "   "

Reply:
Done